### PR TITLE
DDF-3179 UndeliveredMessagesMbean operations can be executed regardless of the system hostname value

### PR DESCRIPTION
**Pull request by roelens8**
#### What does this PR do?
This PR fixes a bug that would prevent the UndeliveredMessagesMbean from being registered if the system hostname was not localhost. For more Information see: https://codice.atlassian.net/browse/DDF-3179

-Changed the Artemis broker name to "artemis"
-The Artemis ObjectName created in the UndeliveredMessagesMbean now uses "artemis" as the broker name instead of the system hostname
-Added missing dependencies to fix sdk example routes
#### Who is reviewing it? 
@ryeats 
@harrison-tarr 
@jrnorth 
(please choose AT LEAST two reviewers that need to approve the PR before it can get merged)
#### Choose 2 committers to review/merge the PR. 
(please choose ONLY two committers from below, delete the rest)
@clockard
@jaymcnallie
#### How should this be tested? (List steps with links to updated documentation)
-Run a full build, all itests should pass.
-Change the system hostname to something other than "localhost", verify the UndeliveredMessagesMbean is registered (no errors should show up in the logs)
-Use a client to send messages to the DLQ, and verify messages can be displayed in the UndeliveredMessages UI.
#### What are the relevant tickets?
[DDF-3179](https://codice.atlassian.net/browse/)
#### Checklist:
- [N/A] Documentation Updated
- [N/A] Update / Add Unit Tests
- [N/A] Update / Add Integration Tests
#### Review Comment Legend:
- ✏️ (Pencil) This comment is a nitpick or style suggestion, no action required for approval. This comment should provide a suggestion either as an in line code snippet or a gist. 
- ❓ (Question Mark) This comment is to gain a clearer understanding of design or code choices, clarification is required but action may not be necessary for approval.
- ❗ (Exclamation Mark) This comment is critical and requires clarification or action before approval.

### CODE REVIEW

File: distribution/sdk/artemis-sandbox/pom.xml [hunk 1]
```diff
@@ -55,7 +55,45 @@
             <artifactId>artemis-jms-client</artifactId>
             <version>${artemis.version}</version>
         </dependency>
+        <dependency>
```
Comment (clockard) [inline distribution/sdk/artemis-sandbox/pom.xml]: This dependency change seems unrelated to the other changes. Just wanted to make sure it is needed.

Comment (roelens8) [inline distribution/sdk/artemis-sandbox/pom.xml]: It is unrelated to fixing the bug. 
However, there is another issue with the sdk examples routes not being able to run due to these missing dependencies.

**Comment by roelens8** [comment]
build now

**Comment by harrison-tarr** [comment]
🔥  👍

**Comment by roelens8** [comment]
build now

**Comment by harrison-tarr** [comment]
successfully heroed.
